addon/components/connect.js

```diff
@@ -53,18 +63,14 @@ export default (stateToComputed, dispatchToActions=() => ({})) => {
       },
 
       handleChange() {
-        const redux = this.get('redux');
-
-        const props = stateToComputed.call(this, redux.getState(), this.getAttrs());
-
-        const notifyProperties = Object.keys(props).filter(name => {
-          return this.get(name) !== props[name];
-        });
-
-        if (notifyProperties.length > 0) {
-          run.join(() => {
-            notifyProperties.forEach(name => this.notifyPropertyChange(name));
-          });
+        Ember.warn(
```

Comment:
There's a subtle change here in that consumers who were overriding this method and calling the original with super (e.g. if they wanted to do more work here) will not have that ability anymore. I doubt anyone would have been doing that, because you would have to write something like:

```
export default connect(stateToComputed, dispatchToActions)(Ember.Component.extend({
...
})).extend({
  handleChange(...args) {
    // Your custom code here...
    this._super(...args);
  }
});
```